lms/envs/common.py

```diff
@@ -3043,15 +3051,37 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 CERT_NAME_LONG = "Certificate of Achievement"
 
 #################### OpenBadges Settings #######################
+# Full guide for setting up OpenBadges available here:
+# https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/enable_badging.html
```

Comment:
I'd move this to the `toggle_description` of `FEATURES['ENABLE_OPENBADGES']`.  What do you think?